i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-commercial-extensions.md

```diff
@@ -140,6 +140,19 @@ Release date: `December 16, 2021`
 
 ## Centreon BAM
 
+### 21.10.3
+
+Release date: `May 30, 2023`
+
+#### Bug fixes
+
+- [Banner] Fixed the display of empty skeleton.
+- [Install] Fixed an issue preventing the module installation when the 24x7 timeperiod parameter was not set.
+- [Monitoring] Fixed an issue in the BA monitoring redirection.
+- [Server] Fixed an issue in the KPI configuration export process to Remote Servers that caused the KPIs' states to be reset at each export of the configuration.
+- [Server] Fixed an issue that prevented the children Business Activities from being displayed on Remote Servers.
+- [Widget] Fixed an issue that made links to Business Activities broken on **BAM - Live Business Activity Tree**.
+
```

Comment:
Just checking.
From the board I see that https://centreon.atlassian.net/browse/MON-14460 and https://centreon.atlassian.net/browse/MON-12299  are embdeded but not reported here. Is it normal?
Not sure also to which issue this RN refers to - [Server] Fixed an issue in the KPI configuration export process to Remote Servers that caused the KPIs' states to be reset at each export of the configuration.

Reply:
added https://centreon.atlassian.net/browse/MON-14460 that was missing
https://centreon.atlassian.net/browse/MON-12299 corresponds to L153